docs/modules/ROOT/pages/userstories-EP.adoc

```diff
@@ -0,0 +1,20 @@
+= User stories Solid CRS-EP
+:description: The user stories for Solid CRS: the Erfgoedpod project.
+:sectanchors:
+:url-repo: https://github.com/netwerk-digitaal-erfgoed/solid-crs
+:page-tags: nde-erfgoed
+:imagesdir: ../images
+:sectnums:
+
+As an admin of a heritage institution, I want to apply for certain collections/objects that I want to digitally lend from another heritage institution, so that I can display them in our own catalog even though we are not the owner .
+As an admin of a heritage institution, I want to approve or dismiss the loan request from another heritage institution, so that I have control over the collections and objects of our institution that are lent. 
+As a user of Solid CRS, I want to copy the unique URL of an object or collection, so that I can refer to the exact object/collection when pasting the URL.
+As an admin of a heritage institution, I want to have an overview of the requests that I sent out, so I don’t request the same objects/collections twice.
+As an admin of a heritage institution, I want to be informed when my loan request gets approved, so I know when the requested loan items are added to our heritage page in Solid CRS.
```

Comment:
the requested loan items will never be automatically added when the loan request was accepted. the requesting institution will receive an approval confirmation, after which they can choose to 'import' the requested collection into their own catalog.

the choosing part is optional, can be done automatically when retrieving the inbox entries after the institution logs in for instance. the point is that accepting the request does not trigger the transfer/copying of the requested collection.